### ISSUE TITLE
Training Loss Function

### ISSUE THREAD

**Issue by melonking32** [castorini/rank_llm#27]
What is the loss function you use during training, LM loss, or ranknet loss？

**Comment by ronakice** [comment]
We train our models with the standard teacher forcing objective.
